docs/earthfile/builtin-args.md

```diff
@@ -32,8 +32,10 @@ The following builtin args are available
 | `EARTHLY_TARGET_TAG` | The tag part of the canonical reference of the current target. Note that if the target has no [canonical form](../guides/target-ref.md#canonical-form), the value is an empty string. | For the example above, the tag would be `john/work` |
 | `EARTHLY_TARGET_TAG_DOCKER` | The tag part of the canonical reference of the current target, sanitized for safe use as a docker tag. This is guaranteed to be a valid docker tag, even if no canonical form exists, in which case, `latest` is used. | For the example above, the docker tag would be `john_work` |
 | `EARTHLY_GIT_HASH` | The git hash detected within the build context directory. If no git directory is detected, then the value is an empty string. Take care when using this arg, as the frequently changing git hash may be cause for not using the cache. | `41cb5666ade67b29e42bef121144456d3977a67a` |
+| `EARTHLY_GIT_SHORT_HASH` | The first 8 characters of the git hash detected within the build context directory. If no git directory is detected, then the value is an empty string. Take care when using this arg, as the frequently changing git hash may be cause for not using the cache. | `41cb5666` |
 | `EARTHLY_GIT_ORIGIN_URL` | The git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. Please note that this may be inconsistent, depending on whether an HTTPS or SSH URL was used. | `git@github.com:bar/buz.git` or `https://github.com/bar/buz.git` |
 | `EARTHLY_GIT_PROJECT_NAME` | The git project name from within the git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. | `bar/buz` |
+| `EARTHLY_GIT_COMMIT_TIMESTAMP` | The timestamp of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `1626881847` |
```

Comment:
Can you say a little more about the format of the timestamp? Is that unix seconds?

Reply:
I expanded that to: `The timestamp, as unix seconds, of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string.`, plus it has the example `1626881847`.